### PR TITLE
[#151] Remove jquery from doc

### PR DESCRIPTION
**Pull request by adrienluitot**
Use vanilla js instead of jQuery for multiple things in the doc like on ready, toggleSlide and to "manually" add classes.
Issue : #151

### CODE REVIEW

**Review by bryanbraun — APPROVED** [review]
Just tested this out and it works like a charm! 😍 

Thanks @adrienluitot!!

File: docs/scripts.js [hunk 1]
```diff
@@ -1,12 +1,31 @@
-$(document).ready(function() {
-  var preEls = $('pre');
+function slideExampleCode(e) {
+  e.preventDefault();
 
-  $('.example-code-link').click(function(e) {
-    e.preventDefault();
-    $(this).parent().next().slideToggle();
-  });
+  let slide = e.target.parentElement.nextElementSibling;
```
Comment (XhmikosR) [inline docs/scripts.js:4]: You shouldn't use ES6+ in docs since old browsers are supported too.

Comment (adrienluitot) [reply docs/scripts.js:4]: I can edit it if you want but it seems supported event with old browsers (after IE 8).

**Comment by XhmikosR** [comment]
It's better to keep everything ES5 because there are bugs
https://stackoverflow.com/questions/2356830/what-browsers-currently-support-javascripts-let-keyword

If/when the supported browsers change in core, then the docs could be
updated. Otherwise it's inconsistent at least.

That's my personal opinion on the matter and what I follow in all my
projects. It doesn't make sense to use let here and have the core use var.

**Comment by adrienluitot** [comment]
@XhmikosR Okay I see, I didn't understand you were talking about `let`.
Hmm if it's just that I think it's better if someone can do it directly without PR, if I do it I will need to make a new PR it's kinda annoying no ?

**Comment by bryanbraun** [comment]
@adrienluitot, @XhmikosR, good questions.

AnchorJS itself supports IE9+, so it probably makes sense for the docs to align with that. It's a quick fix, so I'll just push it up straight to master. 👍

**Comment by adrienluitot** [comment]
Okay perfect thanks ! :D

**Comment by bryanbraun** [comment]
Updated in 552e41adf3f6db8400ec7839ae2f649d20020f16. 👍

**Pull request merged**